### DISCUSSION TITLE
Suggestion: Resw file use filename langeuage identifier rather than folder

### DISCUSSION THREAD

**Discussion by DJDQfff** [unoplatform/uno#18801] [Ideas]
**Uno default project use different folder to place different language resw file.**
    ![PixPin_2024-11-15_17-38-17](https://github.com/user-attachments/assets/7a8417cf-d5ca-46f0-af89-3e9fd3051da9)

**But latest visualstudio updated its resource explorer . it is better to use filename rather folder.**
![PixPin_2024-11-15_17-42-12](https://github.com/user-attachments/assets/0d4bfc95-1e13-4a12-8a40-3a19ffbd521e)

**For new resource explorer, different resw file in different folders is reagrded as different resource**
![PixPin_2024-11-15_17-38-05](https://github.com/user-attachments/assets/b769d4b7-611e-444d-86f9-1c202bc7565b)
![PixPin_2024-11-15_17-57-47](https://github.com/user-attachments/assets/67b28a16-37a5-4680-a35f-55fd3f3e8260)

**But if they are in same folder, it will be same resource.**

![PixPin_2024-11-15_17-56-25](https://github.com/user-attachments/assets/91445eab-2396-414f-bd2b-8901ebf88441)

**Comment by jeromelaban** [comment]
That is an interesting suggestion! I don't think uno supports this qualifier yet, though, but it's worth looking into it. Let's track in https://github.com/unoplatform/uno/issues/18807.
